index.md

```diff
@@ -125,6 +127,20 @@ Back in the terminal:
   1 passing (9ms)
 ```
 
+Set up a test script in package.json:
+
+```json
+"scripts": {
+    "test": "./node_modules/mocha/bin/mocha"
```

Comment:
npm script implicitly have `node_modules/.bin` in `$PATH`. It's enough to just write `"mocha"` here